src/Extensions/ElementalPageExtension.php

```diff
@@ -25,6 +25,17 @@ class ElementalPageExtension extends ElementalAreasExtension
         'ElementalArea',
     ];
 
+    /**
+     * The delimiter to separate distinct elements in indexed content.
+     *
+     * When using the getElementsForSearch() method to index all elements in a single field,
+     * a custom delimiter can be used help to avoid false positive results for phrase queries.
+     *
+     * @config
+     * @var string
+     */
+    private static $delimiter_for_indexed_elements = ' ';
```

Comment:
What's a use case for this to not be a space?

I can't really think of what we'd want to do something like put a dash `-` between element content in search results?

If we do retain this configurable var, could we rename it to `$search_index_element_delimiter`?

Reply:
I initially had it as ellipses but @michalkleiner rightly pointed out that may be unexpected for some developers. In my projects I will want to use an ellipses so that search results show a clear separation between content from separate elemental blocks.

Reply:
OK the ellipses use case makes sense.  Yes rename thanks